helm/grafana/values.yaml

```diff
@@ -16,6 +16,58 @@ pgClusterRecoveryTest:
     name: grafana-postgresql-recovery-test
     namespace: monitoring
 
+#-- Grafana postgresql database, using CNPG
+postgresqlCluster:
+  enabled: true
+  name: grafana-postgresql
+  instances: 2
+  storage:
+    size: 2Gi
+  image:
+    registry: gsoci.azurecr.io
+    repository: giantswarm/postgresql-cnpg
+    tag: 17.7
+  #-- For IRSA access
+  serviceAccount:
+    annotations: {}
+  #-- For customers with their own object storage
+  customAccessKeys:
+    use: false
+    endpointURL: ""
+    region: null
+    accessKeyId: ""
+    secretAccessKey: ""
+  #-- For Azure object storage access
+  #   storageAccount:
+  #     name: storageAccountName
+  #     key: accountName
+  #   storageKey:
+  #     name: storageKeyName
+  #     key: accountKey
+  azureCredentials: {}
+  backup:
+    enabled: false
+    useIRSA: true
+    #-- Path to the object storage where the backups will be stored
+    # ⚠️ Needs to be different from the recovery.destinationPath
+    destinationPath: ""
+    schedule: "0 0 * * * *" # every hour
+    retentionPolicy: 30d
+    #-- Bucket CR config
+    bucket:
+      create: false
+      name: "giantswarm-installationName-grafana-postgresql"
+      installation: "installationName"
```

Comment:
I do not see this being used I only see `.Values.postgresqlCluster.bucket`, but I think it does make sense to have it under `postgresqlCluster.bucket`

Reply:
But the value defined here is `.Values.postgresqlCluster.backup.bucket`